src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -99,68 +112,151 @@ const ChannelImage = ({
     return <FastImage source={{uri: mainPicture}} style={styles.image} />;
   };
 
-  // ANON PM CHANNEL IMAGE
-  if (type === BaseChannelItemTypeProps.ANON_PM)
+  const renderMyPostNotificationSubImage = () => {
+    if (!isCommentExists)
+      return (
+        <View style={[styles.postNotificationImage, styles.myPostNotificationImageContainer]}>
+          <FastImage source={FeedIcon} style={styles.postNotificationIcon} />
+        </View>
+      );
+
+    if (isAnonymousCommenter)
+      return (
+        <ChannelAnonymousSubImage anonPostNotificationUserInfo={anonPostNotificationUserInfo} />
+      );
+
+    return (
+      <FastImage source={{uri: postNotificationPicture}} style={styles.postNotificationImage} />
+    );
+  };
+
+  // SIGNED OR ANON PM CHANNEL IMAGE
+  if (type?.includes('PM')) {
     return (
       <View>
-        <FastImage source={{uri: mainPicture}} style={styles.image} />
-        <View style={[styles.postNotificationImage, styles.anonPmNotificationImageContainer]}>
-          <FastImage source={ChatIcon} style={styles.postNotificationIcon} />
+        {renderChatMainImage()}
+        <View
+          style={[
+            styles.postNotificationImage,
+            isAnonymousTab ? styles.anonPmNotificationImageContainer : styles.backgroundDarkBlue
+          ]}>
+          <FastImage source={ChatIcon} style={styles.chatIcon} />
         </View>
       </View>
     );
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for rendering a PM channel image has been updated to check if the `type` includes 'PM'. This change might affect the behavior of the component if there are other types that include 'PM' in their names. Make sure this change doesn't introduce bugs.
```diff
-  if (type === BaseChannelItemTypeProps.ANON_PM)
+  if (type?.includes('PM')) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->